QamarDeen/QamarDeen/Model/Generated/Reading+CoreDataClass.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  Reading+CoreDataClass.swift
+//  QamarDeen
+//
+//  Created by Mazyad Alabduljaleel on 12/25/16.
+//  Copyright © 2016 Batoul Apps. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+@objc(Reading)
+public class Reading: NSManagedObject {
+
+}
+
+// MARK: Calculations
+
+extension Reading {
+    
+    var ayahCount: Int {
```

Comment:
I actually spent a few minutes contemplating the logic, then realized how it works 😄  Migrated from the old code.